.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Thank you for sending a PR !
+
+Please perform the following checks and tick mark all the points.
+
+- [ ] If its a new page, it does not already exist in the repo.
```

Comment:
"its" should have an apostrophe here. See http://www.bristol.ac.uk/arts/exercises/grammar/grammar_tutorial/page_13.htm